README.md

```diff
@@ -43,6 +47,19 @@ $ npm install --save fuse.js
 $ yarn add fuse.js
 ```
 
+**CDN**
+
+Available on CDN via [jsDelivr](https://cdn.jsdelivr.net/npm/fuse.js/dist/).
+
+### Explanation of Different Builds
+
+In the [`dist/` directory of the NPM package](https://cdn.jsdelivr.net/npm/fuse.js/dist/) you will find many different builds of Fuse.js. Here's an overview of the difference between them:
+
+|                       | UMD         | CommonJS       | ES Module (for bundlers) | ES Module (for browsers) |
+| --------------------- | ----------- | -------------- | ------------------------ | ------------------------ |
+| **Full**              | fuse.js     | fuse.common.js | fuse.esm.js              | fuse.esm.browser.js      |
```

Comment:
I'm not understanding why `fuse.esm.js` might be needed. Bundlers can use `fuse.esm.browser.js`, right?

Reply:
I added the explanation [here](https://github.com/krisk/Fuse/blob/bundler/dist/README.md)

Does it clarify things?

Reply:
Not quite. I looked at both [`dist/fuse.esm.js`](https://github.com/krisk/Fuse/blob/5d76a678a21c7df1ddd11d2290afd3055a599fc9/dist/fuse.esm.js) and [`dist/fuse.esm.browser.js`](https://github.com/krisk/Fuse/blob/5d76a678a21c7df1ddd11d2290afd3055a599fc9/dist/fuse.esm.browser.js), there isn't much difference that could be used for tree shaking (there is a single default export in both). The only difference I saw is fuse.esm.js is transpiled, while browser one is not. When people are using bundlers, they often use transpilers along with it. So we shouldn't need a `fuse.esm.js` (and rename `fuse.esm.browsers.js` to `fuse.esm.js`), unless I'm missing something.

Reply:
Slightly late to mention, but having a `fuse.esm.min.js` would be nice to have.
Thanks for working on it though! 😻

Reply:
I can add it as part of v5.2.0